web/viewer.html

```diff
@@ -209,9 +209,18 @@
               <span data-l10n-id="spread_even_label">Even Spreads</span>
             </button>
 
-            <div class="horizontalToolbarSeparator spreadModeButtons"></div>
+            <div class="horizontalToolbarSeparator"></div>
+            
+            <button id="ltrReadingDirection" class="secondaryToolbarButton readingDirection" title="Set reading direction from left to right" tabindex="68" data-l10n-id="ltr_reading_direction">
```

Comment:
The id should be `readingDirectionLtr` instead, with the `data-l10n-id`s updated accordingly as well.

Reply:
Also, this button should have the `toggled` CSS class set as well.